backend/src/main/java/com/engineering/thesis/backend/model/Appointment.java

```diff
@@ -9,6 +9,10 @@
 
 @Data
 @Entity(name = "appointments")
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
For most of our models we didn't have this 4 addnotations. it's shorting code in many places.

Reply:
Resolved